metadata_finds/metadata_finds.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```

Comment:
@psmatsinhe this script will need some documentation or a user guide to explain on all the steps for the partners.

Reply:
Yes, we need to specify how to run, some kind of SOP

Reply:
SOP can be found here: https://docs.google.com/document/d/1Kj7IbXzVyWimnDtNYEzFWlJaPFj6nbEGASnYuCWM-dc/edit